.github/workflows/automatic-update.yml

```
@@ -0,0 +1,47 @@
+name: Update Grafana Version
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
```

```suggestion
    - cron: "0 8 * * *"
```

might get more attention this way, no need to do it in the night